semperfiwebdesign/all-in-one-seo-pack
# Make "All in One SEO Pack" string untranslatable

Issue #2432

## Proposed changes

This makes the string "All in One SEO Pack" untranslatable again. We decided to do this in order to preserve our brand name/product name across all languages.

## Types of changes

- Improves existing functionality
- Improves existing code

## Checklist


## Testing instructions

Check which strings have been changed and then check these in both the free and premium version of the plugin to see if all still look good.

**You'll have to build Pro by combining the PR in free and Pro.**

## Further comments

PR in Pro can be found here - https://github.com/semperfiwebdesign/aioseop-pro/pull/498

@michaeltorbert ping me once these changes have been pushed to translate.wordpress.org so I can check whether the string is no longer translatable.
